Plugins/NativeServerPlugin/src/directx_multi_peer_conductor.cpp

```diff
@@ -195,14 +201,10 @@ void DirectXMultiPeerConductor::DisconnectFromServer() {}
 
 void DirectXMultiPeerConductor::ConnectToPeer(int peer_id)
 {
-	auto it = connected_peers_.find(peer_id);
-	if (it != connected_peers_.end())
+	auto peer = SafeAllocatePeerMapEntry(peer_id);
```

Comment:
the implementation that was here would always fail, as when you are connecting to a client there is not yet a message from the client, therefore there is no entry here yet. by using the new SafeAllocatePeerMapEntry we create an entry if needed, or use the existing if found